codespell_lib/data/dictionary.txt

```diff
@@ -3936,6 +3936,7 @@ cavaets->caveats
 ccahe->cache
 ccale->scale
 ccertificate->certificate
+ccertificates->certificates
```

Comment:
Add ccertificated->certificated and ccertification->certification

Reply:
6b5626ca558be44bc06f48e61e85c44dff74bd06